svc/pkg/game/ops/version-validate/src/lib.rs

```diff
@@ -718,14 +718,17 @@ async fn handle(
 					}
 				}
 
+				let network_mode = unwrap!(LobbyRuntimeNetworkMode::from_i32(
+					docker_config.network_mode
+				));
+				let host_networking_enabled =
+					std::env::var("RIVET_HOST_NETWORKING").map_or(false, |v| v == "1");
```

Comment:
does this break tests?

Reply:
Is there a case where someone is running a cluster with testing but not host networking? Should those tests be disabled if the env var isnt set?

Reply:
We can assert that host networking is enabled when testing is on.

Reply:
So in that case it shouldn't break any tests

Reply:
We don't assert this atm. It needs to be written.

Reply:
Oh i see what you mean